example/example-sequential-bigmap-local-vs-rpc-pack.ts

```diff
@@ -10,7 +10,7 @@ async function example() {
     try {
         const tezos = new TezosToolkit(rpc);
 
-        const contractAddress = 'KT1Hrp5i3P5BUATvkP7scdqY5PHBmYyz3CNA';
+        const contractAddress = 'KT1VBn1rTZ4FLz43hC9xugn56K9mSM2DQQXJ';
```

Comment:
Not the right contract, should be `KT1VqHMv2j61NSNKBPumRNjh8dDksTmnbGjs` I think

Reply:
Yes, that is the correct replacement. Replaced and verified it works.